### PR TITLE
Cleanup READMEs

### PR DESCRIPTION
**Pull request by DaughterOfMars**
# Description of change

Some suggestions that got missed in the README PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,44 +13,42 @@ modules: `client` and `wallet`.
     - [Dependencies](#dependencies)
 - [Getting Started](#getting-started)
     - [Install the IOTA SDK](#install-the-iota-sdk)
-    - [Usage](#usage)
-        - [Client](#client)
-        - [Wallet](#wallet)
-        - [Examples](#examples)
-    - [API Reference](#api-reference)
+- [Client](#client)
+- [Wallet](#wallet)
```
Comment (lucas-tortora) [inline README.md]:
```suggestion
- [Client](#client-usage)
- [Wallet](#wallet-usage)
```

File: bindings/nodejs/README.md [hunk 2]
```diff
@@ -2,13 +2,12 @@
...
-- [Requirements (only for building the binary yourself)](#requirements-only-for-building-the-binary-yourself)
+- [Requirements](#requirements)
 - [Getting Started](#getting-started)
     - [Install the IOTA SDK](#installing-the-iota-sdk)
-    - [Usage](#usage)
-      - [Client](#client)
-      - [Wallet](#wallet)
-      - [Examples](#examples)
+- [Client](#client)
+- [Wallet](#wallet)
```
Comment (lucas-tortora) [inline bindings/nodejs/README.md]:
```suggestion
- [Client](#client-usage)
- [Wallet](#wallet-usage)
```

File: bindings/python/README.md [hunk 3]
```diff
@@ -7,11 +7,10 @@ Python binding to the [iota-sdk library](/README.md).
 - [Requirements](#requirements)
 - [Getting Started](#getting-started)
     - [Install the IOTA SDK](#install-the-iota-sdk)
-    - [Usage](#usage)
-        - [Client](#client)
-        - [Wallet](#wallet)
-        - [Examples](#examples)
-    - [API Reference](#api-reference)
+- [Client](#client)
+- [Wallet](#wallet)
```
Comment (lucas-tortora) [inline bindings/python/README.md]:
```suggestion
- [Client](#client-usage)
- [Wallet](#wallet-usage)
```

**Review by lucas-tortora — COMMENTED** [review]
just a minor correction on some links, looks good to me :)

**Review by lucas-tortora — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**